en/scripting/modules/import-map.md

```diff
@@ -0,0 +1,90 @@
+# 导入映射（实验性）
```

Comment:
English when merged?

Reply:
Yes, when this pr is merged, I will add it to the list of pending translations for the issue #5638.